src/libraries/FeeVaultUpgrader.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {Utils} from "src/libraries/Utils.sol";
+import {RevShareCommon} from "src/libraries/RevShareCommon.sol";
+import {IOptimismPortal2} from "@eth-optimism-bedrock/interfaces/L1/IOptimismPortal2.sol";
+import {IProxyAdmin} from "@eth-optimism-bedrock/interfaces/universal/IProxyAdmin.sol";
+import {IFeeVault} from "src/interfaces/IFeeVault.sol";
+
+/// @notice Library for upgrading and configuring fee vaults for RevShare
+library FeeVaultUpgrader {
+    /// @notice The creation code for OperatorFeeVault
```

Comment:
Please add a comment regarding which commit in the monorepo this was taken from (it should be the one corresponding to the audit).